### ISSUE TITLE
ggdotchart adjust dot size

### ISSUE THREAD

**Issue by xkcococo** [kassambara/ggpubr#411]
Greetings author,

I am really interested in this package and want to apply it to my own research. It's awesome!
I only have one question: Is it possible to adjust dot size base on dot value in this function? (ggdotchart)
I want to use a bigger dot to stand for a larger value.

Thanks in advance!

**Comment by kassambara** [comment]
Yes — pass a **column name** to `size=` so it becomes a size aesthetic (bigger dot = larger value), and tune the range with `scale_size()`. (`dot.size=` only sets a single fixed size.)

``` r
suppressWarnings(suppressPackageStartupMessages(library(ggpubr)))
df <- data.frame(name = c("A","B","C","D","E"), val = c(2,5,8,12,20))
# Map size to a column (a size aesthetic); tune with scale_size(). dot.size= is a fixed size.
ggdotchart(df, x = "name", y = "val", size = "val",
           sorting = "descending", rotate = TRUE, ggtheme = theme_pubr()) +
  scale_size(range = c(2, 9))
```

![](https://i.imgur.com/ZUOsoCQ.png)

Verified: dots grow A→E with the value.

Closing as resolved — reopen if anything's unclear. 🙌
